scanner/config.go

```diff
@@ -12,6 +12,36 @@ import (
 	"github.com/bitrise-io/go-utils/sliceutil"
 )
 
+const otherProjectType = "other"
+
+type scanResultStatus string
```

Comment:
what's the point of defining the result as strings, you do not print the status.
What about something like

```
type Status int

const (
	NotDetected Status = iota
	DetectedWithErrors
	Detected
)
```

Reply:
fixed